### PR TITLE
Add readme title, description, demo, local setup, technologies and cr…

### PR DESCRIPTION
**Pull request by ivonne-hernandez**
# [Mi Voz, Mi Voto] 🗳

- [x] Feature PR [🧩]
- [ ] Refactor PR [⚙️]
- [ ] BugFix PR [🐞]
- [ ] User Interface PR [🖍]
- [ ] Error Handling PR [⚠️]
- [ ] Testing PR [🧑‍💻]

## **What (if any) features are you implementing?**
 - Creating a read me.

## **What (if anything) did you refactor?**
 - (NA)

## **Were there any issues that arose?**
 - We still don't have our FE and BE connected so I wasn't able to create a video demo-ing our State Election Reminders form or emails that are supposed to be sent to the subscriber from our BE.

## **Is there anything needs to be prioritized/errors to be addressed?**
 - (NA)

## **Goals? Next steps?**
 - Get deployed FE, and deployed BE connected.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,22 @@
...
+- `cd mi-voz-mi-voto`
+- `npm install`
+- `npm start` to start the app on localhost:3000
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Technologies
+- React
+- React Router
+- Cypress
```
Comment (elisebeall) [inline README.md:23]: I would also include here:
- [React Intl](https://formatjs.io/docs/react-intl/)
- [Cypress Code Coverage](https://github.com/cypress-io/code-coverage)
- [Instrument CRA](https://github.com/cypress-io/instrument-cra)

File: README.md [hunk 2]
```diff
@@ -1,70 +1,22 @@
...
-In the project directory, you can run:
+## Application demo:
+https://user-images.githubusercontent.com/86989161/154371168-c8426fa6-9d72-4b74-90dc-42e4555156bf.mov
 
-### `npm start`
+## Local Setup
+- `git clone` this repository
+- `cd mi-voz-mi-voto`
+- `npm install`
+- `npm start` to start the app on localhost:3000
```
Comment (elisebeall) [inline README.md:13]: You could also add here: 

`npm run dev` to start the app on localhost:3000, start cypress, and compile a code coverage report
`open coverage/lcov-report/index.html` to view code coverage report in the browser

File: README.md (hunk 1 above)
Comment (elisebeall) [reply README.md:23]: Context API

**Review by elisebeall — CHANGES_REQUESTED** [review]
Few fun little additions!

File: README.md (hunk 1 above)
Comment (ivonne-hernandez) [reply README.md:23]: Thank you for reminding me to add these.

File: README.md (hunk 2 above)
Comment (ivonne-hernandez) [reply README.md:13]: Absolutely. I've added this as well.

**Review by elisebeall — APPROVED** [review]
Thanks for adding these updates!

**Pull request merged**